docs/Testing.rst

```diff
@@ -24,11 +24,14 @@ This is a common way to do it.
 Pull Request Testing for spack-c2sm on GitHub
 ---------------------------------------------
 
-To test a PR, create a comment ``launch jenkins``.
+To test a PR, create a comment ``launch jenkins``. This will trigger a Jenkins job that will test the PR.
```

Comment:
For which machines are "launch jenkins" tests?

Reply:
Made it more clear